nix/pkgs/core.nix

```diff
@@ -14,19 +14,46 @@ rec {
     set -euxo pipefail
     export RUST_BACKTRACE=1
 
+    hc-standard-test
+    hc-slow-test
+    hc-wasm-test
+  '';
+
+  hcStandardTests = writeShellScriptBin "hc-standard-test" ''
+    set -euxo pipefail
+    export RUST_BACKTRACE=1
+
+    # limit parallel jobs to reduce memory consumption
+    export NUM_JOBS=8
+    export CARGO_BUILD_JOBS=8
+
+    # run all the non-slow cargo tests
+    cargo build --features 'build_wasms' -p holochain_wasm_test_utils
```

Comment:
the `holochain_wasm_test_utils` crate doesn't have a feature called `build_wasms`. it only has these:

```toml
[features]
default = []
build = []
only_check = []
```

looks like we want `build` here